scss/fonts.scss

```diff
@@ -0,0 +1,7 @@
+* {
```

Comment:
This should stay in `styles.scss` file as it is global

Reply:
Do you mean: 
* {
  font-family: 'Roboto', sans-serif;
} 
because in a previous comment you tell me to put it in a fonts.scss file (or i understand that at least), or just: 
*{
}

Reply:
It was just for the `text-big` and the others :P the one with the `*` should stay in styles.scss